src/groups/bmq/bmqex/bmqex_executor.t.cpp

```diff
@@ -115,15 +115,15 @@ class StatExecutor {
     void post(FUNCTION f) const
     {
         ++d_statistics_p->d_postCallCount;
-        f();
+        bslmf::MovableRefUtil::access(f)();
```

Comment:
Why is this necessary? Is `FUNCTION` being used as a `MovableRef<T>`?

Reply:
Yes, it was one if the compilation errors